pyp2rpm/metadata_extractors.py

```diff
@@ -135,7 +135,10 @@ def name_convert_deps_list(self, deps_list):
     @property
     def has_pth(self):
         """Figure out if package has pth file """
-        return "." in self.name
+        if virtualenv is None:
+            return "." in self.name
+        else:
```

Comment:
You are right of course, but I think it is more obvious what return values we expect in other modules when it is there.